proposal-template.wit.md

```diff
@@ -26,7 +26,7 @@ More rigorous specification details for the implementer go here, if needed.
 /// Short description
 ///
 /// Explanation for developers using the API.
-api-function-one: function() -> api-type-one
```

Comment:
Why do these templates live in the proposal itself?   Shouldn't it be instances of the templates that live here?  (apologies I've not be following the proposal template stuff).  

i.e. shouldn't the templates get renamed when a proposal is started?

Reply:
i guess we should rename them when we start using them.
this PR is meant to be the minimum ci fix.

i personally prefer to remove them until we actually start using them: https://github.com/WebAssembly/wasi-threads/pull/23

Reply:
No, I was just curious.  This pr seems fine.